WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -482,39 +482,36 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
+	htmlArr[idx++] = "<table class='TopRow'>";
 	htmlArr[idx++] = "<tr id='";
 	htmlArr[idx++] = DwtId.getListViewItemId(DwtId.WIDGET_ITEM_FIELD, this._view, task.id, ZmItem.F_ITEM_ROW_3PANE);
 	htmlArr[idx++] = "'>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
+	if (task.pComplete) {
+		htmlArr[idx++] = "<td class='ZmTaskHeaderImage'>";
```

Comment:
There are some css rules already present with that class name (zm.css near line 2358):
```
.ZmTaskProgress {
	@FullWidth@
}

.ZmTaskProgress DIV {
	display:block;
	height:100%;
}
```
Do we need these now?